src/Sake.Engine/Builder/BuilderBase.cs

```diff
@@ -55,7 +57,13 @@ public void StartingTarget(string name)
             {
                 CallTarget(dependency);
             }
-            Log.Info(string.Format("Target \x1b-\x0f{0}\x1b-\x07", name));
+

```

Comment:
Instead of changing this method in sake source, override it as well in the `_use-teamcity.shade` file

```

functions
  @{
    bool IsTeamCity { get { return true; } }

    class TeamCityLog : Sake.Engine.Logging.ILog
    {
        public void Info(object value) { }
        public void Warn(object value) { }
        public void Verbose(object value) { }
        public void Error(object value) { }
    }

    private string _previousTarget;

    public void StartingTarget(string name)
    {
        if (!IsTeamCity) { base.StartingTarget(name); return ;}

        foreach (var dependency in Targets[name].Dependencies)
        {
            CallTarget(dependency);
        }

        if (_previousTarget != null)
        {
            Console.WriteLine(string.Format("##teamcity[blockClosed name='{0}']", _previousTarget));
        }
        _previousTarget = name;
        Console.WriteLine(string.Format("##teamcity[blockOpened name='{0}']", name));
    }
  }

@{
  if (IsTeamCity) { Log = new TeamCityLog(); }
}
```